src/site/content/en/fast/priority-hints/index.md

```diff
@@ -563,50 +563,9 @@ With the fix in Chrome 95 and the enhancement for priority hints, we hope that d
-
-You can also enable the **Experimental Web Platform Features** flag in Chrome to test the feature.
-
-<figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/NNupmBrjCHQ6maB4U6NA.png", alt="A screenshot of Chrome's experimental web features toggled on under the hood. This screen is accessible by navigating to chrome://flags in Chrome.", width="800", height="260" %}
-</figure>
-
-### Using the trial
-
-The following is a comparison between a set of images and a script file loaded with different priorities. Note that the default priorities for images and scripts are applied on Chrome 94. On the other hand, the priority set through the importance attribute is applied when tested on Chrome 96.
-
-Specifying an origin trial token in an HTML `<meta>` element is often the quickest and most convenient method. In the case of Priority Hints, however, this may result in hints being overlooked for resources loaded early on during startup. For Priority Hints to work optimally, you should specify the origin trial token in an `Origin-Trial` HTTP response header:
-
-```http
-Origin-Trial: {Replace with origin trial token}
-```
-
-Once your application is configured to send that header, you can then use Priority Hints in a variety of scenarios in your HTML as needed:
-
-```html
-<!-- Lower the priority of script file -->
-<script src="script.js" importance="low"></script>
-
-<!-- Alter the priority of images -->
-<img src="Background.jpg" width="400" importance="low">
-<img src="Sunset.jpg" width="400" importance="high">
-
-<!-- Note that importance="auto" is the default based on the spec if not specified -->
-<img src="Flower.jpg">
-```
-
-<figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/mmjf9CVbs0wqLSq4Y6Hg.png", alt="A screenshot of Chrome's DevTools open for a page in Chrome 94 with a largely unstyled document. The document includes a few images, as well as a script asset. Because this is Chrome 94 and Priority Hints were not implemented at that time, the priority hints take no effect, and the browser assigns the default priorities.", width="800", height="499" %}
-  <figcaption>Priority on Chrome 94 - stable version</figcaption>
-</figure>
-
-<figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/4fIgpdvgryzGkfYyrAbt.png", alt="A screenshot of the same document with Chrome's DevTools open, only in Chrome 96. This time, the Priority Hints take effect when set for resources, and the effect is shown in the priority column at right.", width="800", height="499" %}
-  <figcaption>Priority on Chrome 96 - Canary</figcaption>
-</figure>
+## History
+Priority hints was first experimented with in Chrome as an origin trial in 2018 and then again in 2021 using the `importance` attribute. The interface has since changed to `fetchpriority` for HTML and `priority` for JavaScript's fetch() API as part of the web standards process.
```

Comment:
```suggestion
Priority hints was first experimented with in Chrome as an origin trial in 2018 and then again in 2021 using the `importance` attribute. The interface has since changed to `fetchpriority` for HTML and `priority` for JavaScript's Fetch API as part of the web standards process.
```

Reply:
If you meant to refer only to the `fetch()` method then say `fetch()` method instead of Fetch API.